.travis.yml

```diff
@@ -26,15 +26,12 @@ env:
         - SYMFONY_DEPRECATIONS_HELPER=weak
 
 before_install:
-    - if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then git fetch origin "refs/pull/$TRAVIS_PULL_REQUEST/merge"; else git fetch origin "$TRAVIS_BRANCH"; fi;
-    - if [[ "$TRAVIS_COMMIT" != `git rev-parse FETCH_HEAD` ]]; then echo "Pull request or branch commit hash has changed, aborting!"; exit 1; fi;
```

Comment:
Somehow: they have $TRAVIS_COMMIT hash that do not exist anymore in the main repos. I suspect this is related to github doing merges all the time in `refs/pull/*/merge`, eveytime we merge something in the target branch.